recipe/meta.yaml

```
@@ -1,17 +1,17 @@
 {% set name = "unicorn" %}
-{% set version = "1.0.3" %}
+{% set version = "2.0.1.post1" %}
 
 package:
   name: {{ name|lower }}
```

Let's start with this
```suggestion
  name: {{ name|lower }}-split
```